metabase/metabase
# Dashboard verifications

Closes https://github.com/metabase/metabase/issues/48954

### Description
Adds Moderation Status to the Dashboard entity in places like get Dashboard, recents and search.

### How to verify

1. Navigate to a dashboard, go to the actions menu and click "Verify this Dashboard"
2. There should be a verified Icon in the dashboard header
3. navigate to the collections page and you should see a verified icon in the table.
4. Open the command palette. There should be a verified icon next to the dashboard in recents and search

### Demo

_Upload a demo video or before/after screenshots if sensible or remove the section_

### Checklist

- [x] FE Tests have been added/updated to cover changes in this PR
- [x] BE Tests have been added/updated to cover changes in this PR
